### PR TITLE
Add initial support for kelkoo integration

### PR DESCRIPTION
**Pull request by icazevedo**
**What problem is this solving?**

Adds Kelkoo integration support

### CODE REVIEW

File: pixel/head.html [hunk 1]
```diff
@@ -1,11 +1,24 @@
...
+      return
+    }
+
+    if(!countryCode) {
+      console.error('Warning: No Kelkoo product country identifier is defined. Please configure it in the apps admin.');
       return
     }
 
-    // Add script here
+    _kkstrack = {
```
Comment (klzns) [inline pixel/head.html]:
```suggestion
   window._kkstrack = {
```

File: pixel/head.html [hunk 2]
```diff
@@ -1,11 +1,24 @@
-<script>
-  (function() {
-    var appId = "{{settings.gtmId}}";
-    if (!appId) {
-      console.error('Warning: No APP NAME ID is defined. Please configure it in the apps admin.');
+<script async="true" type="text/javascript" src="https://s.kk-resources.com/leadtag.js" ></script>
```
Comment (klzns) [inline pixel/head.html]: Check if it's possible to add this snippet on the end of the <body> tag. If it's possible you can change the name of this file to `body.html` and this snippet will be included on the end of the <body> tag.

File: docs/README.md [hunk 3]
```diff
@@ -1,56 +1,17 @@
...
-Next, you can **add media** (either an image of a GIF) if possible, so that users can better understand how the app works in practice. 
+```
+vtex install vtex.kelkoo-pixel@0.x
+```
 
-![Media Placeholder](https://user-images.githubusercontent.com/52087100/71204177-42ca4f80-227e-11ea-89e6-e92e65370c69.png)
+Open your admin in the Apps section:
 
-## Configuration
+https://YOURSTORENAME.myvtex.com/admin/apps/
```
Comment (klzns) [inline docs/README.md]: I think you should use the doc model that comes from the template

File: .all-contributorsrc [hunk 4]
```diff
@@ -1,14 +0,0 @@
-{
-  "projectName": "pixel-app-template",
```
Comment (klzns) [inline .all-contributorsrc:2]: You can keep this file just change this line to the name of the repo

**Review by klzns — CHANGES_REQUESTED** [review]

File: pixel/head.html (hunk 2 above)
Comment (icazevedo) [reply pixel/head.html]: The [docs](https://www.kelkoogroup.com/pt/suporte/suporte-a-comerciantes/manual-integration/#) say it is. I'll do it

**Review by klzns — APPROVED** [review]

**Pull request merged**